opendrift/readers/basereader/__init__.py

```diff
@@ -309,14 +309,13 @@ def plot(self, variable=None, vmin=None, vmax=None,
         if variable is not None:
             rx = np.array([self.xmin, self.xmax])
             ry = np.array([self.ymin, self.ymax])
-            data = self.__get_variables_derived__(variable, self.start_time,
-                                      rx, ry)
+            data = self.get_variables(variable, self.start_time, rx, ry)
```

Comment:
This method is not guaranteed to be available. It will also give different result depending whether your're using a structured or unstructured reader (same as before). The only guaranteed methods, which is also always work in the same way, are `get_variables_interpolated` and `get_variables_interpolated_xy`.

Reply:
Ok, but then also the reader.plot() method needs to be rebuilt. So this would be a quick temporal fix.

Reply:
Sounds good 👍

Reply:
It is desirable here to show the reader background data in native resolution ("rotated raw pixels"). With interpolated data, some tricks will be needed to achieve this.

Reply:
And actually that means using nearest interpolation, which I believe is not available(?) Thus perhaps we can use some method based on top of `get_variables` But adapted to the three types of readers.